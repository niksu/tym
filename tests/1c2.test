hellom1123_AS12_1() :- a(), b(), c(), d().
c() :- a(). a(b).
