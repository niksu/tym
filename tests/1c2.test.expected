input contents |hellom1123_AS12_1() :- a(), b(), c(), d().
c() :- a(). a(b).
|
(no query given)
stringed file contents (size=60, remaining=1540)
|hellom1123_AS12_1() :- a(), b(), c(), d().
c() :- a().
a(b).|
